client/tokenized-express-checkout/blocks/components/express-checkout-component.js

```diff
@@ -118,29 +125,53 @@ const ExpressCheckoutComponent = ( {
 		onReady( event );
 	};
 
-	// The Cart & Checkout blocks provide unified styles across all buttons,
-	// which should override the extension specific settings.
-	const withBlockOverride = () => {
-		const override = {};
-		if ( typeof buttonAttributes !== 'undefined' ) {
-			override.buttonHeight = Number( buttonAttributes.height );
-		}
+	const checkoutElementOptions = useMemo( () => {
+		// The Cart & Checkout blocks provide unified styles across all buttons,
+		// which should override the extension specific settings.
+		const withBlockOverride = () => {
+			const override = {};
+			if ( typeof buttonAttributes !== 'undefined' ) {
+				override.buttonHeight = Number( buttonAttributes.height );
+			}
+			return {
+				...buttonOptions,
+				...override,
+			};
+		};
 		return {
-			...buttonOptions,
-			...override,
+			...withBlockOverride(),
+			...adjustButtonHeights( withBlockOverride(), expressPaymentMethod ),
+			...getPaymentMethodsOverride( expressPaymentMethod ),
 		};
-	};
+	}, [ expressPaymentMethod, buttonAttributes, buttonOptions ] );
+
+	useEffect( () => {
+		if ( ! isPreview || onElementsReadyCalled.current ) {
+			return;
+		}
+
+		const handle = setTimeout( () => {
+			if ( ! onElementsReadyCalled.current ) {
+				setShowFallbackButton( true );
+			}
+		}, FALLBACK_BUTTON_WAIT_TIME );
+
+		return () => clearTimeout( handle );
+	}, [ isPreview, onElementsReadyCalled ] );
```

Comment:
```suggestion
	}, [ isPreview ] );
```

The value returned by `useRef` shouldn't be a hook dependency.